### PR TITLE
2023.3.1 Change Log

### PR DESCRIPTION
**Pull request by Bojan-Svirkov**

### CODE REVIEW

File: src/modules/ROOT/pages/changelog/202303/01.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      "refs": [
+        ""
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "charts and reporter elements utilizing aggregation erroring out.",
+        "The issue was caused because the sqserver driver does not yet support offloading of aggregation steps."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202303/01.json]: sqserver -> sqlserver

File: src/modules/ROOT/pages/changelog/202303/01.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      "how": [
+        "by not updating the title when creating a block."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1170/commits/3ee9fd5cddabdb39ca7eafd90086c12a96c427f9"
+      ]
+    },
+    {
+      "what": [
+        "data and time input boxes not being response and resulting in being cut off on smaller screens."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202303/01.json]: not being response and -> not being responsive and

**Review by tjerman — CHANGES_REQUESTED** [review]

File: src/modules/ROOT/pages/changelog/202303/01.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        ""
+      ],
+      "why": [
+        ""
+      ],
+      "refs": [
+        ""
+      ]
+    }
+  ],
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202303/01.json]: Don't forget to remove this before merging

**Review by tjerman — APPROVED** [review]
lgtm; good job

File: src/modules/ROOT/pages/changelog/202303/01.json [hunk 4]
```diff
@@ -26,9 +23,29 @@
...
+      "why": [
+        "to give user more freedom in defining error handling and to know where and on which step the error occurred."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1088"
+      ]
+    },
+    {
+      "what": [
+        "http prefixes to all URL input values across {PRODUCT_NAME} if it is not provided by the user."
```
Comment (tjerman) [inline src/modules/ROOT/pages/changelog/202303/01.json:42]: i'm quite sure it's https not http; fact-check with the author

**Review by tjerman — APPROVED** [review]
See that comment; else lgtm
good job

File: src/modules/ROOT/pages/changelog/202303/01.json (hunk 4 above)
Comment (Bojan-Svirkov) [reply src/modules/ROOT/pages/changelog/202303/01.json:42]: Firstly I wrote it https but after checking with the author he corrected me that it needs to be http. So all good.

**Pull request merged**